### ISSUE TITLE
Has this been past a patent lawyer?

### ISSUE THREAD

**Issue by digulla** [twitter/innovators-patent-agreement#22]
I'm worried that this will just create a new playing field for lawyers:
- Companies suing/threatening inventors to withdraw their claim so they can sell the patent anyway
- Expensive law suits after a company filed bankruptcy and the "assets" are bought by a patent troll who just chooses ignores this contract (their business model is pushing the law, after all) so the inventor would have to sue them

So my question is: Has a patent lawyer had a look at this?

Also keep in mind that everything is well until the patent becomes a multi-billion dollar product at which time greed takes over. What are the other loop holes of this contract? How can it be abused?

**Comment by benltwitter** [comment]
Yes. This is not legal advice, but I have been a practicing patent lawyer for longer than I would care to admit.  Mark Lemley and various others looked at it before we posted it to Github.  

But I think you do hit your head upon an important point of this enterprise: to come up with an agreement that has as few loopholes as possible.  As is evident from the standards arena, this is not an easy task.

**Comment by tarul** [comment]
One of my loophole concerns is the "voluntarily participated" language. Crazy scenario:
- Company A sells a patent portfolio to a troll along with an obligation to participate in enforcing the patent claims
- Troll sues people and Company A assists them in the lawsuits
- Company A claims they didn't voluntarily participate because they were obligated to participate in the suits due to their patent sale agreement

Is that voluntarily once its a step removed, i.e. they voluntarily sold the patents with an obligation but then were obligated to participate in the suit.

What if participation is limited to a subpoena? They could still be forced to testify but can't work with lawyers or provide any resources to build an offensive patent infringement case since that is not covered in the exclusions.

**Comment by benltwitter** [comment]
Yes, that's a tricky point. If Company A unloading patents to an entity that is effectively a shell to be used against another, that is merely offensive use of a patent through proxy.  But mere transfer of a set of patents to another entity probably shouldn't bind that entity to everything that entity does.  Distinguishing between the two cases is difficult.

**Comment by tarul** [comment]
I don't think binding is all that bad. It puts the onus on companies to ensure that whenever they transfer patents they adhere to the IPA restrictions (i.e. those are incorporated into the transfer agreements). Even companies that are not using the IPA will have to be aware that by not incorporating IPA restrictions when they transfer patents (i.e. agreeing to assist in prosecuting offensive patent suits), they could open themselves up to suits from IPA companies.

Most importantly it closes a potential loophole. Its only for the patents they transfer & agree to help prosecute, not for any other patents the transferee chooses to prosecute. 

That said, its worthwhile identifying & discussing areas where that "binding" could have unintended consequences. At this time, I can't imagine any areas where we'd want a transferring company to not incorporate IPA restrictions on the transfer.
